platform-emf/src/main/java/com/softicar/platform/emf/form/EmfForm.java

```diff
@@ -24,6 +24,7 @@
 
 	private final IEmfFormFrame<R> frame;
 	private final R tableRow;
+	private final DomDiv formContainer;
```

Comment:
hmm, would be great if we would not have to add this artificial container div

Reply:
I added this container because it simplifies to layout custom EmfForm implementations

Reply:
to simplify this, I would propose such a class:
```java
public class EmfFormBridge<R extends IEmfTableRow<R, ?>> extends DomDiv implements IEmfForm<R> {

	protected EmfForm<R> form;

	public EmfFormBridge(IEmfFormFrame<R> frame, R tableRow) {

		this.form = new EmfForm<>(frame, tableRow);
	}

	@Override
	public IEmfFormFrame<R> getFrame() {

		return form.getFrame();
	}

	@Override
	public R getTableRow() {

		return form.getTableRow();
	}
	...
```